postgres/package.json

```diff
@@ -45,15 +46,17 @@
               "user": "postgres",
               "password": "postgres",
               "database": "postgres"
-            }
+            },
+            "schema_evolution": "auto"
           }
         },
         "postgres": {
           "impl": "@cap-js/postgres",
           "dialect": "postgres",
           "vcap": {
             "label": "postgresql-db"
-          }
+          },
+          "schema_evolution": "auto"
```

Comment:
I think it's only required to be set here. The profiles refer to this config via `kind` property.